workshops/static/amy_utils.js

```diff
@@ -0,0 +1,134 @@
+/* A file with functionality that is enabled on each page. */
+
+function bulk_email() {
+    /*
+    Look at all selected <input type="checkbox" email="example@example.org">
+    tags on the webpage and join email addresses using comma as a separator
+    (i.e. "first@example.org,second@example.org").
```

Comment:
I think we need to look for more general solutions (ref #934) -- but maybe later